front-end/src/components/List/List.scss

```
@@ -2,4 +2,8 @@
 
 .list {
   list-style-type: none;
+  &Header {
+    margin-bottom: 22px;
```

```suggestion
    margin-bottom: 24px;
```
Use the defined spacing